src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -51,8 +50,8 @@ def get(self, request, project):
         # no longer feature gated and is added as an IntegrationFeature
         result["integrations"] = [
             serialize(i, request.user)
-            for i in filter(
-                lambda i: i.has_feature(IntegrationFeatures.STACKTRACE_LINK), integrations
+            for i in list(
```

Comment:
Note to self: I think this cannot be simplified further because we may end up with "collection modified during iteration"